solr/webapp/web/js/angular/controllers/cloud.js

```diff
@@ -424,7 +424,7 @@ var nodesSubController = function($scope, Collections, System, Metrics) {
               // These are the cores we _expect_ to find on this node according to the CLUSTERSTATUS
               var cores = nodes[node]['cores'];
               if (! cores) {
-                cores = {};
+                cores = [];
```

Comment:
Since there's the `cores[coreId]` use at line 433, wondering if leaving `cores = {}` here and "if not empty" qualifying the `cores.sort` call at line 470 might be an alternative?

Reply:
Thanks, I've changed it to do `cores.sort` when "if not empty".